### ISSUE TITLE
facet_grid() doesn't work for geom_barplot()

### ISSUE THREAD

**Issue by asteriske** [yhat/ggplot#515]
The following produces a 5x7 facet grid in R, but not in Python. See http://stackoverflow.com/questions/38649657/facet-grid-not-working-in-ggplot-for-python

```
from ggplot import *

diaa = diamonds[['cut','color','table']]
diab = diaa.groupby(['cut','color']).quantile([x/100.0 for x in range(0,100,5)])
diab.reset_index(inplace=True)
diab.columns = ['cut','color','p','stats']
ggplot(diab,aes(x='p',weight='stats')) + geom_bar() + facet_grid('color','cut')
```

```
/home/mccarthy/.pyenv/versions/2.7.12/envs/plotting/lib/python2.7/site-packages/ggplot/ggplot.pyc in __repr__(self)
    117 
    118     def __repr__(self):
--> 119         self.make()
    120         # this is nice for dev but not the best for "real"
    121         if os.environ.get("GGPLOT_DEV"):

/home/mccarthy/.pyenv/versions/2.7.12/envs/plotting/lib/python2.7/site-packages/ggplot/ggplot.pyc in make(self)
    600                                 facet_filter = facetgroup[self.facets.facet_cols].iloc[0].to_dict()
    601                                 for k, v in facet_filter.items():
--> 602                                     mask = (mask) & (df[k]==v)
    603                                 df = df[mask]
    604 

TypeError: 'NoneType' object has no attribute '__getitem__'
```

Apparently 'df' doesn't have a value k?

**Comment by glamp** [comment]
a188dd4ebd1712b70bd489d188b022adef5d8c9a

**Comment by glamp** [comment]
fixed in 0.11.1
